src/ProjNet/CoordinateSystems/Projections/MapProjection.cs

```diff
@@ -244,6 +244,14 @@ public ProjectionParameter GetParameter(string name)
             return _Parameters.Find(name);
         }
 
+        /// <summary>
+        /// Adds a parameter to the parameter list
+        /// </summary>
+        internal void AddParameter(string name, double value)
```

Comment:
This is not needed to get the solution to compile. The parameters need to be present when the class is constructed, so adding stuff here will not do any good.